W-15308521 multiruntime implement duke (#292)
mulesoft/docs-code-builder: modules/ROOT/pages/int-create-integrations.adoc

. From *Quick Actions*, click *Develop an Integration*:
+
image::develop-integration-option-mat.png["Develop an Integration link highlighted in the Getting started section"]

> . Provide a project name and location to the *Develop an Integration* form.

Reviewer: ```suggestion
. Specify a project name and location in the *Develop an Integration* form.
```
A minor suggestion. Optional.
